src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -82,6 +82,10 @@
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
         <service id="Symfony\Component\Serializer\Normalizer\ObjectNormalizer" alias="serializer.normalizer.object" />
+        
+        <service id="serializer.normalizer.flatten_exception" class="Symfony\Component\Serializer\Normalizer\FlattenExceptionNormalizer">
```

Comment:
You should move this configuration to PHP.

Reply:
Thanks! I will do it.